### PR TITLE
Sets Remote URL at connection level instead of pull

### PR DESCRIPTION
**Pull request by lordofthejars**
#### Short description of what this resolves:

Sets Remote URL at connection level instead of pull

#### Changes proposed in this pull request:

-
-
-

**Fixes**: #632

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Just a suggestion - if you don't have details to fill-in don't leave the template - it makes unnecessary noise in the PR description.

File: docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java [hunk 1]
```diff
@@ -712,10 +718,6 @@ public void pullImage(String imageName) {
 
             PullImageCmd pullImageCmd = this.dockerClient.pullImageCmd(image.getName());
 
-            if (this.cubeConfiguration.getDockerRegistry() != null) {
-                pullImageCmd.withRegistry(this.cubeConfiguration.getDockerRegistry());
```
Comment (bartoszmajsak) [inline docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java]: So before we were pulling with registry and now we set registry globally? Would that work for all images?

Comment (lordofthejars) [inline docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java]: No, before we were just attaching the repo url to the image name. It is a way to append the url of registry to image name programmatically. But this should be already provided by user on docker-compose.

Comment (bartoszmajsak) [inline docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java]: Ok, so docker client executor and cubeConfiguration is per image? What if I have 1 image available in my private repo and other from the docker hub? Are we able to handle it?

**Comment by lordofthejars** [comment]
Yes if you define correctly the docker-compose with correct urls.

El dl., 20 de març 2017 a les 10:41, Bartosz Majsak (<
notifications@github.com>) va escriure:

**Pull request merged**